source/driverHandler.py

```diff
@@ -163,7 +91,7 @@ def _paramToPercent(cls, current, min, max):
 		@param max: The maximum value.
 		@type max: int
 		"""
-		return int(round(float(current - min) / (max - min) * 100))
+		return paramToPercent(current, min, max)
 
 	@classmethod
 	def _percentToParam(cls, percent, min, max):

```

Comment:
I kept these because I didn't want to change the interface of the `driverHandler` module. Even these methods are preceded by an underscore, and should be considered private, I can't verify that no external driver is using them. There is also very little cost to keeping them around.

Reply:
But, as driverHandler.Driver inherrits from autoSettingsUtil.autoSettings.AutoSettings, it will already have these methods from there. You're not breaking anything when removing it. Or am I just missing something essential here?

Reply:
Yes, you are right. I'll move the documentation too.